README.md

```diff
@@ -3,21 +3,19 @@ Code Runner: A tool to help run your assignments
 Code Runner is a web application to run code pertaining to your assignment using user provided configurations.
 
 ## Features
-There are two roles in the application - Professor and Student.
-#### Professor
-- Professor provides course specific configurations for creating suitable docker environment.
-- Course specific configurations include specifying operating system and its version, programming language and compiler.
+There is one role in the application - Student.
 #### Student
 - Students can submit their assignments and view the output through a web page.
-- They can also enter the optional command line arguments that are required to run their assignment.
+- They can enter the assignment working directory, commands to compile and run the assignment.
```

Comment:
as this is a separate point, you shouldn't start the sentence with "They". If you want to do that then it should be a subpoint.

Reply:
Okay. I have made them as sub points.